app/tests/api/util/test_string_utils.py

```diff
@@ -19,37 +19,3 @@ def test_blank_for_null():
     assert blank_for_null("hello") == "hello"
     assert blank_for_null(4) == "4"
     assert blank_for_null(["a", "b"]) == "['a', 'b']"
-
-
-def test_mask_pii():
```

Comment:
Did this test get lost when the function was moved? I think it's valuable as the regexp `TIN_RE` is complex.

Reply:
probably, good catch i'll bring it back, sorry for losing it!

Reply:
Good catch, I actually introduced a bug that I wouldn't have caught if I didn't bring it back. Take another look to see if it looks right. It's in a new file called test_pii.py

Reply:
One small change in behavior to note, I didn't end up casting the input to a string if there wasn't a regex match. It broke things to change types of stuff in the record that log handlers weren't expecting to have changed.